cukes-graphql/src/main/java/lv/ctco/cukes/graphql/facade/GQLResponseFacade.java

```diff
@@ -38,24 +37,21 @@ public void doRequest() throws Exception {
         requestFacade.initNewSpecification();
     }
 
-    private Callable<ResponseWrapper> doRequest(final HttpMethod method) {
-        return new Callable<ResponseWrapper>() {
-            @Override
-            public ResponseWrapper call() throws Exception {
-                final RequestSpecification requestSpec = requestFacade.value();
+    private Callable<Response> doRequest(final HttpMethod method) {
```

Comment:
GQLResponseFacade and RestResponseFacade look pretty much the same. To my believe they should share the same features (e.g. load-runner) and the same step sequence while performing an HTTP call.